### DISCUSSION TITLE
Empty `resolvers` array property found in `buildSchema` options!

### DISCUSSION THREAD

**Discussion by Hasilt** [MichalLytek/typegraphql-nestjs#20] [Q&A]
I'm getting Empty `resolvers` array property found in `buildSchema` options! error message.

This my `TypeGraphQLModule` import in `AppModule` (using Nest7).  resolvers is the default import from generated resolvers
```
 TypeGraphQLModule.forRootAsync({
      inject: [ConfigService],
      useFactory: async (configService: ConfigService) => {
        const graphqlConfig = configService.get<GraphqlConfig>('graphql');
        return {
          buildSchemaOptions: {
            resolvers: resolvers,
            emitSchemaFile: true,
          },
          cors: true,
          sortSchema: graphqlConfig.sortSchema,
          autoSchemaFile:
            graphqlConfig.schemaDestination || './src/schema.graphql',
          debug: graphqlConfig.debug,
          path: graphqlConfig.endPoint,
          dateScalarMode: 'isoDate',
          playground: graphqlConfig.playgroundEnabled,
          context: ({ req }) => ({ req }),
         
        };
      }
    }),
```

**Comment by MichalLytek** [comment]
Dude, this is a `typegraphql-prisma` repo, not about `TypeGraphQLModule` nor `typegraphql-nestjs`. 

Your config is invalid, you're mixing NestJS options, go read the manual first before asking such questions.

**Comment by MichalLytek** [comment]
Moved into a proper repo. 

Please stop spamming with tons of reports, I don't have hours of time to help everybody.

**Reply by Hasilt** [reply]
Sorry for posting on the `typegraphql-prisma` repo. at the time of posting `typegraphql-nestjs` repo didn't had a discussion section. 

Sorry for bothering you. Let me try again with default configs in the repo.
